runtime/indent/stylus.vim

```diff
@@ -0,0 +1,127 @@
+" Vim indent file
+" Language: Stylus
+" Maintainer: Marc Harter
+" Last Change: 2010 May 21
+" Based On: sass.vim from Tim Pope
+"
+if exists("b:did_indent")
+  finish
+endif
+unlet! b:did_indent
+let b:did_indent = 1
+
+setlocal indentexpr=GetStylusIndent()
+setlocal indentkeys=o,O,*<Return>,},],0),!^F
+unlet! b:undo_indent
+let b:undo_indent = 1
```

Comment:
still wrong. Please check other indent scripts for how the undo_indent var is used